### PR TITLE
Update version Swagger UI to 3.28.0

### PR DESCRIPTION
**Pull request by zkatemor**
Upgrade to the latest release of swagger-ui:
https://github.com/swagger-api/swagger-ui/releases

This will fix the problem with empty fields: https://github.com/rswag/rswag/issues/328
Fix from swagger-ui: https://github.com/swagger-api/swagger-ui/pull/5830

### CODE REVIEW

**Comment by zkatemor** [comment]
@BookOfGreg

**Comment by tmaier** [comment]
They released already the version 3.26.1 and 3.26.2, which fixes a security issue. Could you update your pull request?

**Comment by floriandejonckheere** [comment]
Swagger UI 3.27 has been released with some new features and fixes.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Once you have an API that can describe itself in Swagger, you've opened the trea
 
 |Rswag Version|Swagger (OpenAPI) Spec.|swagger-ui|
 |----------|----------|----------|
-|[master](https://github.com/rswag/rswag/tree/master)|3.0.3|3.23.11|
+|[master](https://github.com/rswag/rswag/tree/master)|3.0.3|3.26.0|
```
Comment (BookOfGreg) [inline README.md]:
```suggestion
|[master](https://github.com/rswag/rswag/tree/master)|3.0.3|3.28.0|
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Update swagger-ui version to 3.23.11 [#239](https://github.com/rswag/rswag/pull/239)
 - Rails constraint moved from < 6.1 to < 7 [#253](https://github.com/rswag/rswag/pull/253)
 - Swaggerize now outputs base RSpec text on completion to avoid silent failures [#293](https://github.com/rswag/rswag/pull/293)
+- Update swagger-ui version to 3.26.0
```
Comment (BookOfGreg) [inline CHANGELOG.md]:
```suggestion
- Update swagger-ui version to 3.28.0
```

**Pull request merged**